templates/invoices_app/invoice_detail.html

```diff
@@ -87,6 +87,7 @@ <h6 class="mb-3">Details:</h6>
     <span><button type="button" class="btn btn-info">Edit info</button></span>
     {% endif %}
     {% if is_AP %}
+    <span><a href="{% url 'invoice-history' pk=invoice.id %}" ><button type="button" class="btn btn-primary">History</button></a></span>
```

Comment:
Why do you guys use this html structure here?
`<a><button>...</button></a>`

In terms of semantic, anchor tags(<a>) are used for links that redirects you to another page and button(<button>) are used to trigger actions or functions o some sort of "behavior".